cluster-provision/k8s/1.32/k8s_provision.sh

```diff
@@ -351,23 +348,6 @@ chmod -R 777 /var/local/kubevirt-storage/local-volume
 # Setup selinux permissions to local volume directories.
 chcon -R unconfined_u:object_r:svirt_sandbox_file_t:s0 /mnt/local-storage/
 
-# copy network addons operator manifests
-# so we can use them at cluster-up
-cp -rf /tmp/cnao/ /opt/
-
-# copy whereabouts manifests
-# so we can use them at cluster-up
-cp -rf /tmp/whereabouts/ /opt/
-
-# copy Multus CNI manifests so we can use them at cluster-up
-cp -rf /tmp/multus /opt/
-
-# copy cdi manifests
-cp -rf /tmp/cdi*.yaml /opt/
-
-# copy aaq manifests
-cp -rf /tmp/aaq/ /opt/
```

Comment:
In the check-provision I don't see the aaq deployed, at least the [flag is set to false](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/1356/check-provision-k8s-1.32/1884318918537908224#1:build-log.txt%3A1217). Is that tested somewhere else?

Reply:
@Barakmor1 do you know? Maybe we can check that part by just enabling the flag on the provision-check or is there something that speaks against this?